docs/api/crash-reporter.md

```diff
@@ -46,12 +46,11 @@ The `crashReporter` module has the following methods:
   * `companyName` String
   * `submitURL` String - URL that crash reports will be sent to as POST.
   * `productName` String (optional) - Defaults to `app.name`.
-  * `uploadToServer` Boolean (optional) - Whether crash reports should be sent to the server
-    Default is `true`.
+  * `uploadToServer` Boolean (optional) - Whether crash reports should be sent to the server. Default is `true`.
   * `ignoreSystemCrashHandler` Boolean (optional) - Default is `false`.
   * `extra` Object (optional) - An object you can define that will be sent along with the
     report. Only string properties are sent correctly. Nested objects are not
-    supported and the property names and values must be less than 64 characters long.
+    supported and the property names and values must be less than 64 characters (when using Windows).
```

Comment:
The run-on sentence isn't new to this PR; but since we're here, we might as well fix it :slightly_smiling_face: 

```suggestion
    supported. When using Windows, the property names and values must be fewer than 64 characters.
```